Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.yml

```
@@ -2618,27 +2146,20 @@ script:
       description: The incident score.
       type: Number
   - arguments:
-    - default: false
-      description: The endpoint ID.
-      isArray: false
+    - description: The endpoint ID.
       name: id
-      required: false
-      secret: false
     - default: true
       description: The endpoint IP address.
-      isArray: false
       name: ip
-      required: false
-      secret: false
-    - default: false
-      description: The endpoint hostname.
-      isArray: false
+    - description: The endpoint hostname.
       name: hostname
-      required: false
+    - default: true
```

```suggestion
    - default: false
```
Only the ***ip*** argument should be default for endpoint command.
For more information, please review the generic end point [docs](https://xsoar.pan.dev/docs/integrations/generic-endpoint-command).

---

I am not sure how this got changed. I did not make these changes to the integration. I have changed it back to false as you suggested.